recipes/liboptv/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: optv
+  version: 0.2.3
+
+source:
+  url: https://github.com/OpenPTV/openptv/archive/0.2.3.tar.gz
```

Comment:
Please, also add ``sha256``
Please see:
https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=sha256#populating-the-hash-field